packages/odyssey-react-mui/src/themes/odyssey/typography.ts

```diff
@@ -28,37 +28,37 @@ export const typography: ThemeOptions["typography"] = {
     letterSpacing: 0,
   },
   h1: {
-    fontWeight: Tokens.FontWeightBold,
+    fontWeight: Number(Tokens.FontWeightBold),
```

Comment:
We can wrap it with `Number` when defining it as `StrippedBorderRadiusBase`. Maybe we should call it `NumericalBorderRadiusBase`?